### ISSUE TITLE
Why block scrolling up?

### ISSUE THREAD

**Issue by rolandschuetz** [micho/jQuery.preventMacBackScroll#11]
Hello,

I couldn't find any reason why to block scrolling. Could you please add a few infos about this.

From my understanding moving the touchpad upwards doesn't result in any page change, it only causes a end-of-website bounce effect. This should be only done when the user set it up that way. I for example want to block page changes, but not this bounce effect.

Thanks,
Roland

**Comment by micho** [comment]
I did this to prevent divs from overscrolling when reaching the top. The scenario is when you are scrolling in one div inside another div which also scrolls.

**Comment by rolandschuetz** [comment]
As mentioned above, this is not really what the plugin described what it does. I would add a config for that.

**Comment by micho** [comment]
You're right, would you like to send a PR with defaults that can disable that setting?

**Comment by rolandschuetz** [comment]
Sorry, I’m currently super busy. When I find some time I will.

—
Roland Schütz
Web Developer & UX Designer
P: +43 676 45 16 731

Gesendet mit Sparrow (http://www.sparrowmailapp.com/?sig)

Am Mittwoch, 02. September 2015 um 17:26 schrieb Pablo Villalba:
